app/assets/javascripts/discourse/app/lib/text.js

```diff
@@ -157,7 +157,6 @@ export function excerpt(cooked, length) {
         resultLength += element.textContent.length;
       }
     } else if (element.tagName === "A") {
-      element.innerHTML = element.innerText;
```

Comment:
Was the draft preview the only place that was (ab)using this behavior?

Reply:
Yes, I have checked this and I couldn't find any other matches.